update to guava 32.0.0 to resolve CVE-2023-2976

**Describe the bug**
CVE-2023-2976 was reported in https://github.com/google/guava/issues/2575 and appears to be fixed in guava 32.0.0



**SDK version**
java 5.10.8

**Language version, developer tools**
Java 11
**OS/platform**
MacOS Ventura

**Additional context**
Add any other context about the problem here.

Same ...

```
[ERROR] Failed to execute goal org.owasp:dependency-check-maven:8.2.1:aggregate (default-cli) on project brm-parent: 
[ERROR]
[ERROR] One or more dependencies were identified with vulnerabilities that have a CVSS score greater than or equal to '4,0': 
[ERROR]
[ERROR] guava-32.0.0-jre.jar: CVE-2023-2976(6.2)
[ERROR]
[ERROR] See the dependency-check report for more details.
```

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -46,7 +46,7 @@ java {
 
 object Versions {
     const val gson = "2.7"
-    const val guava = "30.1-jre"
+    const val guava = "32.0.1-jre"
     const val ldNanoHttpd = "1.0.0-SNAPSHOT"
     const val okhttpTls = "4.8.1"
 }

